backend/clickhouse/errors.go

```diff
@@ -552,47 +552,43 @@ func (client *Client) QueryErrorHistogram(ctx context.Context, projectId int, qu
 var ErrorObjectsTableConfig = model.TableConfig[modelInputs.ReservedErrorObjectKey]{
 	TableName: ErrorObjectsTable,
 	KeysToColumns: map[modelInputs.ReservedErrorObjectKey]string{
+		modelInputs.ReservedErrorObjectKeyBrowser:         "Browser",
+		modelInputs.ReservedErrorObjectKeyClientID:        "ClientID",
+		modelInputs.ReservedErrorObjectKeyEnvironment:     "Environment",
+		modelInputs.ReservedErrorObjectKeyHasSession:      "HasSession",
+		modelInputs.ReservedErrorObjectKeyOsName:          "OSName",
 		modelInputs.ReservedErrorObjectKeySecureSessionID: "SecureSessionID",
-		modelInputs.ReservedErrorObjectKeyRequestID:       "RequestID",
```

Comment:
Synced with Clickhouse table:
- Added `Browser`, `ClientID`, `HasSession`, `OSName`
- Removed `RequestID`, `SpanId`, `LogCursor`, `Source`, `StackTrace`, `Payload`
- Moved `Event` and `Type` to Group reserved words
- Renamed `URL` -> `VisitedURL`, `Service.Name` -> `ServiceName`, `Service.Version -> ServiceVersion`